masschallenge/semantic-ui-theme
# AC 6676: Horizontally scroll tab sub navigation

Changes introduced in [AC-6676](https://masschallenge.atlassian.net/browse/AC-6676):
- Allow horizontal scrolling for sub navigation 

**How to test:**
- Masquerade as [User](https://test6.masschallenge.org/admin/simpleuser/user/43639/masquerade/)
- Head over to [IL finalists](https://test6.masschallenge.org/dashboard/il-finalist/il-finalist-landing-page/)
- Click [Resources](https://test6.masschallenge.org/nav/resources) on the side nav
- You will then see the top sub nav. You can decrease the size of the window and you will see that the sub nav bar doesn't overflow off the screen and it scrollable with a shadow that indicates which side of the nav bar has more scrollable items

**Locally**
- As Admin demoadmin.masschallenge.org
- Go to [fluent pages](http://localhost:8181/admin/fluent_pages/page/261/change/)
- Add sub nav association nodes along with a sub nav item e.g.
 `home_nav` and sub nav item `room booking`
Note: Add several items  to the same node(about 5)
- Masquerade as user(http://localhost:8181/admin/simpleuser/user/11167/masquerade/)
- You will find several sub nav items. You will then see the top sub nav. You can decrease the size of the window and you will see that the sub nav bar doesn't overflow off the screen and it scrollable with a shadow that indicates which side of the nav bar has more scrollable items
